autoload/neoterm/repl/python.vim

```diff
@@ -0,0 +1,57 @@
+"init python specific settings
+if !exists('g:neoterm_repl_ipy_magic')
+  let g:neoterm_repl_ipy_magic = 0
+end
+
+if !exists('g:neoterm_ipy_cellmark')
+  let g:neoterm_repl_ipy_cellmark = "# %%"
+end
+
+function! neoterm#repl#python#cell()
+    let l:lnum1 = search(g:neoterm_repl_ipy_cellmark, 'Wb') + 1
+    let l:lnum2 = search(g:neoterm_repl_ipy_cellmark, 'W') - 1
+    if l:lnum2 == -1
+      let l:lnum2 = line('$')
+    endif
+    let l:lines = getline(l:lnum1, l:lnum2)
+    call g:neoterm.repl.exec(l:lines)
+endfunction
+
+function! neoterm#repl#python#run()
+  if join(g:neoterm_repl_command) =~ "ipython"
+    let l:cmd = ["%run " . expand('%')]
+  else
+    let l:cmd = ['exec(open("' . expand('%') . '").read())']
+  endif
+  call g:neoterm.repl.instance().exec(add(l:cmd, g:neoterm_eof))
+endfunction
```

Comment:
Although great features, I'd rather keep only one feature per PR.
We could introduce `neoterm#repl#python#exec` on this PR, and when it's merge you can open open a new PR for each of these functions.

Reply:
OK. I will move it to a new branch as soon as I find out how git does it for me.